example.mjs

```diff
@@ -1,28 +1,29 @@
-import { start } from './index.js'
+import fastify from 'fastify'
+import { restartable } from './index.js'
 
-async function myApp (app, opts) {
-  console.log('plugin loaded')
+async function createApp (fastify, opts) {
+  const app = fastify(opts)
```

Comment:
The `createApp` function should be a Fastify plugin and not receive the external fastify constructor.
I liked that part of the design of that module, what's the reason for this specific change?

Reply:
Agreed.